### ISSUE TITLE
Version 1.1 to 1.2 diff

### ISSUE THREAD

**Issue by wilmTap** [jiscdev/analytics-udd#33]
Taking the earliest commits of the UDD to github as 1.1

**Comment by robwynj** [comment]
I want to basically illustrate changes between 1.1 and 1.2 in the most fluent and (for us) automated way possible, for HEIs who will be  referencing this, having been exposed and using UDD 1.1.

**Comment by arc12** [comment]
For the avoidance of doubt:
just to say that I don't think Tribal has any need for the diff, either for internal or customer use.
Cheers, Adam

**Comment by wilmTap** [comment]
I've delved into making such a diff, and I'm afraid it might take some time. While it's easy to do a diff between 1.1 and 1.2 here on github, tens of thousands of line changes in 127 commits and the fact that there are now sixteen files where there were two, make that diff challenging to navigate or understand.

I've tried various other approaches, but the only one that looks presentable to a wide audience involves painstakingly recreating 1.1 by hand in the shape of 1.2, and diff that. [Here's what that looks like for the Course entity](https://github.com/ds10/analytics-udd/pull/3/files?short_path=33a843e#diff-33a843e5bfcdab72419ac5c3d10c7597). If that looks OK, I'll work through all the other entities as well.
